samples/net/cellular_modem/prj.conf

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=y
+CONFIG_NET_L2_PPP=y
```

Comment:
I needed `CONFIG _NET_L2_PPP_OPTION_MRU=y` and `CONFIG_NET_L2_PPP_OPTION_DNS_USE=y` or else PPP negotiation never succeeds.